test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -892,9 +892,9 @@ export default class TabularView extends Component {
                     baselineDataObject.platformsSpecificData;
                 Object.keys(consolidatedDataObject.platformsSpecificData).map(
                     function (key, index) {
-                        consolidatedDataObject.platformsSpecificData[
+                        return (consolidatedDataObject.platformsSpecificData[
                             key
-                        ].relativeComparison = 0;
+                        ].relativeComparison = 0);
```

Comment:
We do not need return in Line 897 and 858

Reply:
so should return null??
because if we don't return anything we will get a warning.

Reply:
Please see https://github.com/adoptium/aqa-test-tools/pull/650#discussion_r867160140